src/log.js

```diff
@@ -159,7 +156,7 @@ export class Log {
       } else if (level == 'WARN') {
         fn = this.win.console.warn || fn;
       }
-      messages.unshift(Date.now() - start, '[' + tag + ']');
+      messages.unshift('[' + tag + ']');
```

Comment:
Should we kill the tag? I find them distracting, but others may disagree.

Reply:
The tag can be helpful when you're scanning through logs in a console. Not sure if people rely on it for debugging. For example, `gulp test --verbose` could benefit from it. I agree it's superfluous in the dev tools window.

I'm fine removing it, but that will involve changing all the calling sites. Okay if that happens in a separate PR?

Reply:
On second thought, I agree that removing the tag is the way to go, since logs will now be visible to publishers, etc. who use the dev tools console.

Reply:
You could guard it by `localDev`

Reply:
@cramforce Done. PTAL.